### PR TITLE
1.1.0 Changes

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**
- Added full screen images
- Rewrote setup, readme, and Usage
- Annotated screenshots

#### Descriptions:
_Add PR descriptions here_

#### Fix:
_Add fix/changes here_

### CODE REVIEW

File: docs/setup.md [hunk 1]
```diff
@@ -17,11 +17,39 @@
...
+    <tr>
+        <td>Playbook Collection</td>
+        <td>A drop-down menu to choose the playbook collection containing the action button and JSON data provider playbooks.</td>
+    </tr>
+    <tr>
+        <td>JSON Data Provider Playbook</td>
+        <td>A playbook to return the JSON data to be rendered in the grid view.</td>
+    </tr>
+    <tr>
+        <td>Enable Grid Action Buttons</td>
```
Comment (cs-dev-chaudhary) [inline docs/setup.md]: "Enable to Grid Action button" is changed to "Grid Action Buttons"

File: docs/usage.md [hunk 2]
```diff
@@ -3,44 +3,82 @@
...
+                    <li><code>grid_data</code>: Contains the list of JSON data to render on grid view. Every record should have a unique IRI (<code>@id</code>) field.</li>
+                    <li><code>grid_columns</code>: Map <code>grid_data</code> variable fields with the column fields of the grid.</li>
+                </ol>
+            </td>
+        </tr>
+    </table>
 
 - The following screenshot shows sample contents of the `grid_data` and `grid_columns` variable
 
     ![](./res/playbook_output.png)
```
Comment (cs-dev-chaudhary) [inline docs/usage.md]: The screenshot is not visible in file view.

File: docs/usage.md [hunk 3]
```diff
@@ -3,44 +3,82 @@
...
 
-- Add JSON to Grid widget to the list view of any module.
+5. Select the playbook that returns the JSON data to be rendered in the grid view.
 
-- Configure the widget by selecting `Playbook Collection` as **Sample - JSON to Grid** and `JSON Data Provider Playbook` as **JSON to Grid** playbook.
+    ![Playbook containing JSON data](./res/edit-view-02.png)
+
+6. Select the checkbox **Action Buttons (No Record Selection)**.
+
+    ![](./res/edit-view-03.png)
```
Comment (cs-dev-chaudhary) [inline docs/usage.md]: Wrong screenshot attached, please check.

File: docs/setup.md (hunk 1 above)
Comment (cs-abhishek-shukla) [reply docs/setup.md]: Changed

File: docs/usage.md (hunk 2 above)
Comment (cs-abhishek-shukla) [reply docs/usage.md]: Removed.

File: docs/usage.md (hunk 3 above)
Comment (cs-abhishek-shukla) [reply docs/usage.md]: Removed

**Comment by cs-dev-chaudhary** [comment]
Approved

File: README.md [hunk 4]
```diff
@@ -10,11 +10,17 @@
 
 - **Applicable**: View Panel and Listing View
 
+- [Release Notes](./widget/release_notes.md)
```
Comment (cs-dev-chaudhary) [inline README.md:13]: Can we change the compatibility to 7.4.3 and later. As we have only tested from 7.4.3 this time.

Comment (cs-abhishek-shukla) [reply README.md:13]: Done

**Pull request merged**